packages/ai-core/src/browser/prompttemplate-contribution.ts

```diff
@@ -34,15 +34,15 @@ export const PROMPT_TEMPLATE_EXTENSION = '.prompttemplate';
 export const DISCARD_PROMPT_TEMPLATE_CUSTOMIZATIONS: Command = Command.toLocalizedCommand({
     id: 'theia-ai-prompt-template:discard',
```

Comment:
We should add a localized label here as well, otherwise, the command ID with Theia AI is still visible.